internal/http/processor.go

```diff
@@ -393,46 +357,46 @@ func (p *processor) ParseTorrentHandler(c *gin.Context) {
 	p.log.Info().Msg("starting to parse season pack torrent")
 
 	if err := json.NewDecoder(c.Request.Body).Decode(&p.req); err != nil {
-		p.log.Error().Err(err).Msgf("error decoding request")
-		c.AbortWithStatusJSON(domain.StatusDecodingError, gin.H{
-			"code":  domain.StatusDecodingError,
-			"error": err.Error(),
+		p.log.Error().Err(err).Msgf(domain.StatusDecodingError.Message())
```

Comment:
If the status type is changed you can change all these to `.Msgf("%s", domain.StatusDecodingError)`

Reply:
you are referring to this right? 😄 
https://github.com/nuxencs/seasonpackarr/pull/148#discussion_r1794199196